driver-core/src/main/java/com/datastax/driver/core/ReplicationStategy.java

```diff
@@ -55,6 +56,7 @@ static ReplicationStrategy create(Map<String, String> replicationOptions) {
             }
         } catch (NumberFormatException e) {
             // Cassandra wouldn't let that pass in the first place so this really should never happen
+            logger.error("Failed to parse replication options: " + replicationOptions, e);
```

Comment:
👍 for leaving the catch block. Maybe that's a bit far-fetched, but if replication factors can ever be something other than integers, we'd want older driver versions to be resilient. If we let the exception propagate it would go all the way to `SchemaParser#refresh`, skipping everything after the keyspace, vs. just having one `KeyspaceMetadata` with missing replication settings.